gradle/plugins/src/main/kotlin/com.hedera.gradle.root.gradle.kts

```diff
@@ -115,19 +115,14 @@ tasks.register("versionAsSnapshot") {
 tasks.register("versionAsSpecified") {
     group = "versioning"
 
-    inputs.property(
-        "newVersion",
-        providers
-            .gradleProperty("newVersion")
-            .orElse(
-                provider {
-                    throw IllegalArgumentException(
-                        "No newVersion property provided! " +
-                            "Please add the parameter -PnewVersion=<version> when running this task."
-                    )
-                }
-            )
-    )
+    inputs.property("newVersion", providers.gradleProperty("newVersion").orElse("INVALID"))
```

Comment:
This should leverage an `orElseThrow()` if available. If `orElseThrow()` is not available then `null` should be returned and the `if` should check for a `null` value instead of a magic string.

Reply:
I'll do the check on `null`

Reply:
Well... the orElse requires either a provider<String!> or a <String!> parameter. So I can use the empty string.